src/state/store/configureStore.js

```diff
@@ -0,0 +1,6 @@
+import { createStore } from "redux";
+import rootReducer from "../reducers/rootReducer";
+
+const store = createStore(rootReducer);
+
+export default store;
```

Comment:
I will merge this now, but since you are now exporting store here, you can import it in your service and avoid sending dispatch through props.